akka-persistence-query/src/main/scala/akka/persistence/query/typed/scaladsl/LoadEventQuery.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.query.typed.scaladsl
+
+import scala.concurrent.Future
+
+import akka.annotation.ApiMayChange
+import akka.persistence.query.scaladsl.ReadJournal
+import akka.persistence.query.typed.EventEnvelope
+
+/**
+ * [[EventsBySliceQuery]] that is using a timestamp based offset should also implement this query.
+ *
+ * API May Change
+ */
+@ApiMayChange
+trait LoadEventQuery extends ReadJournal {
+
+  def loadEnvelope[Event](persistenceId: String, sequenceNr: Long): Future[Option[EventEnvelope[Event]]]
```

Comment:
Wondering if we could just return `Future[EventEnvelope]` and return a failure if not found. 

If someone is calling this method, it's probably the plugin implementer and given the context this will be used it's highly likely that there is such an event in the journal.